cmd/eskip/media_defaults_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2015 Zalando SE
```

Comment:
pls rename this file to mediadefaults_test.go, or move the code to media_test.go, it fits there perfectly.